tgt_grease/enterprise/Model/CentralScheduling.py

```diff
@@ -180,11 +185,31 @@ def determineExecutionServer(self, role):
             str: MongoDB Object ID of server; if one cannot be found then string will be empty
 
         """
-        result = self.ioc.getCollection('JobServer').find({
-            'active': True,
-            'roles': str(role)
-        }).sort('jobs', pymongo.DESCENDING).limit(1)
```

Comment:
Wait, DESCENDING? That should return the most used node, which should always be the same.